app/permissions/ability.rb

```diff
@@ -60,6 +60,7 @@ def initialize(user)
       can?(:create_supply, circle)
     end
     cannot :create_item, Circle do |circle|
+      (circle.working_groups.map { |wg| wg.active_admins.include?(user) and wg.status == 'active'}).none? or
```

Comment:
Works nicely as is! A suggestion, though:
!circle.working_groups.any? { |wg| wg.active_admins.include?(user) && wg.active? })

- collection.any? returns true if the block returns true for at least one item in the collection.
- we use ActiveRecord's enum feature for the status field; I'd prefer to hide value used internally behind a method call, in case we need to change the internal implementation later. Also, it's a little shorter.